modules/olm-updating-operator-catalog-image.adoc

```diff
@@ -32,6 +32,23 @@ For example, for the `podman` CLI:
 ----
 $ REG_CREDS=${XDG_RUNTIME_DIR}/containers/auth.json
 ----
+* If you are working with catalogs from private namespaces on
```

Comment:
Same comment as above, although on second read, it makes more sense now as you have it written. 🤷‍♂️